### PR TITLE
docs(cypress): Update Cypress README Documentation

### PR DESCRIPTION
**Pull request by Gnanasundari24**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [x] Documentation
- [ ] CI/CD

## Description

In this PR, I have updated the README file to include detailed information about Cypress test scenarios and the folder structure . 
    Added detailed instructions for adding new test scenarios to the ConnectorTest directory.
    Included a clear explanation of the folder structure, guiding contributors on where to find specific files and directories related to Cypress tests.
    Emphasized the importance of writing clear test scenarios using Cypress commands for effective testing.

By providing a descriptive title and detailed description, reviewers can quickly understand the purpose of the changes and ensure that the README file enhancements align with the project's goals and standards.

### Additional Changes

- [ ] This PR modifies the API contract
- [ ] This PR modifies the database schema
- [ ] This PR modifies application configuration/environment variables

## Motivation and Context

This update aims to improve the documentation for contributors and provide a clearer understanding of the testing approach.

## Checklist

- [ ] I formatted the code `cargo +nightly fmt --all`
- [ ] I addressed lints thrown by `cargo clippy`
- [ ] I reviewed the submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable

### CODE REVIEW

**Review by jarnura — APPROVED** [review]

**Review by likhinbopanna — APPROVED** [review]

File: cypress-tests/readme.md [hunk 1]
```diff
@@ -1,45 +1,196 @@
...
+}
+```
+
+2.Add the new connector details to the ConnectorUtils folder (including CardNo and connector-specific information).
+
+Refer to Stripe.js file for guidance:
+
+```javascript
+/cypress-tests/cypress/e2e/ConnectorUtils/Stripe.js
+```
```
Comment (SanchithHegde) [inline cypress-tests/readme.md:117]: As an improvement, we can include a hyperlink to the file.
Optional change, and/or can be taken up in a separate PR.

File: cypress-tests/readme.md [hunk 2]
```diff
@@ -1,45 +1,196 @@
...
-    ```
-    npm run cypress 
-    ```
-    To run the tests in headless mode run the following command
-    ```
-    npm run cypress:ci
-    ```
+4. Run Cypress test cases
+
+   To execute a connector create test, ensure the connector details are included in the `creds.json` file. Then, integrate the path in the `command.js` as follows:
```
Comment (SanchithHegde) [inline cypress-tests/readme.md:44]: `command.js` should be `commands.js`?
Here and elsewhere in this doc.

**Review by SanchithHegde — APPROVED** [review]

File: cypress-tests/readme.md (hunk 2 above)
Comment (Gnanasundari24) [reply cypress-tests/readme.md:44]: Sure ill add it in other PR
